ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -154,6 +154,11 @@ trap 'echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_END"' EXIT
 case "${TEST_TYPE}" in
 upgrade-conformance)
     upgrade
+    for FILE in e2e-events.json e2e-intervals.html e2e-intervals.json; do
```

Comment:
Please don't do this here, fix it upstream

Reply:
fix it how?  With JUnit-style timestamps on the filenames?

Reply:
Assuming that's what you want, openshift/origin#26049 will move us to JUnit style timestamp suffixes.

/close